.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 sudo: false
 go:
+- 1.12.x
```

Comment:
Could you then change line 21 to use `1.12` as the go version check?

Reply:
sounds good, ok just updated but will wait for build result